pkg/venafi/cloud/connector.go

```diff
@@ -306,47 +311,46 @@ func (c *Connector) GetPolicy(name string) (*policy.PolicySpecification, error)
 	ps := buildPolicySpecification(cit, info, true)
 
 	// getting the users to set to the PolicySpecification
-	users, error := c.getUsers()
-	if error != nil {
-		return nil, error
+	policyUsers, err := c.getUsers()
+	if err != nil {
+		return nil, err
 	}
-	ps.Users = users
+	ps.Users = policyUsers
 
 	return ps, nil
 }
 
 func (c *Connector) getUsers() ([]string, error) {
 	var usersList []string
-	appDetails, _, error := c.getAppDetailsByName(c.zone.getApplicationName())
-	if error != nil {
-		return nil, error
+	appDetails, _, err := c.getAppDetailsByName(c.zone.getApplicationName())
+	if err != nil {
+		return nil, err
 	}
-	var teams *teams
+	var teamsList *teams
 	for _, owner := range appDetails.OwnerIdType {
 		if owner.OwnerType == UserType.String() {
-			user, error := c.retrieveUser(owner.OwnerId)
-			if error != nil {
-				return nil, error
+			retrievedUser, userErr := c.retrieveUser(owner.OwnerId)
+			if userErr != nil {
+				return nil, userErr
 			}
-			usersList = append(usersList, user.Username)
-		} else {
-			if owner.OwnerType == TeamType.String() {
-				if teams == nil {
-					teams, error = c.retrieveTeams()
-					if error != nil {
-						return nil, error
-					}
+			usersList = append(usersList, retrievedUser.Username)
+		} else if owner.OwnerType == TeamType.String() {
+			if teamsList == nil {
+				teamsList, err = c.retrieveTeams()
+				if err != nil {
+					return nil, err
 				}
-				if teams != nil {
-					for _, team := range teams.Teams {
-						if team.ID == owner.OwnerId {
-							usersList = append(usersList, team.Name)
-							break
-						}
+			}
+			if teamsList != nil {
```

Comment:
We don't need this validation if you already validated that `teamsList` is nil above.